qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -2,9 +2,8 @@
     type = x86_cpu_model
     cpu_model_flags += ",enforce"
     auto_cpu_model = no
-    cpu_model = host
```

Comment:
why drop this line?

Reply:
view from doc string your case test boot VM with cpu = host .

Reply:
This line is useless now since we use other method to get cpu lists. And updated doc string to make it clearer. Thanks.